pytorch_lightning/plugins/environments/lightning_environment.py

```diff
@@ -49,13 +49,15 @@ def creates_processes_externally(self) -> bool:
         """
         return "LOCAL_RANK" in os.environ
 
-    def master_address(self) -> str:
+    @property
+    def main_address(self) -> str:
```

Comment:
feel this is just adding confusion when we rename MASTER_ from os env to main_ locally